test/browser/test/browser-crash.test.ts

```diff
@@ -0,0 +1,15 @@
+import { it } from 'vitest';
+
+it.skip('fails gracefully when browser crashes', async () => {
```

Comment:
I dont know what to do with this test. It reproduces the issue where browser crashes, but if I enable it it will break the pipeline

Reply:
You can add this test to fixtures and create a new test file in test/browser/specs to run the fixture and assert that it fails correctly